docs/docs/services/email/mail-clients.md

```diff
@@ -0,0 +1,136 @@
+    IMAP and send mail, but any received mail will go to your forwarding address and will
+    not be delivered locally.
+
+    If you wish to deliver both locally and forward to another address, find an email
+    client with these features and configure your forwarding there.
+
+## Client Configuration
+
+You can configure your mail client to point to `mail.pydis.wtf`, we support SMTP
+submission on port 465 or 587, leave TLS/SSL settings as your mail client
+defaults.
+
+Your mail client should also support IMAP via the same host.
+
+You can use your username or full email as the IMAP & SMTP username.
+
+You can configure your mail client to send from any of your aliases, e.g.
+`joe@pydis.wtf` or `joe@pydis.com`.
+
+To summarise:
+
+| Configuration Option | Value                |
+|----------------------|----------------------|
+| Mail server address  | `mail.pydis.wtf`     |
+| Inbox protocol       | `IMAP`               |
+| SMTP Port            | `465` or `587`       |
+| SMTP TLS             | Mail client default  |
+| SMTP Username        | `username@pydis.wtf` |
+| SMTP Password        | Your LDAP password   |
+
+If you need any extra help or believe your client requires other settings that
+are not provided by the PyDis mailserver please let us know in `#dev-oops`.
+
+## Sieve
+
+We support rudimentary server-side email filtering with [Pigeonhole
+Sieve](https://doc.dovecot.org/configuration_manual/sieve/pigeonhole_sieve_interpreter/).
+
+Using this, users can perform common mail tasks automatically by writing small
+sieve scripts that are able to act on inbound mail before it reaches a user
+inbox.
+
+!!! note
+
+    Sieve only applies to mail destined for IMAP folders, if mail is being forwarded
+    then sieve scripts will not be executed. It is up to you to implement Sieve scripts
+    or use other inbox rule engines at your destination mailserver if you have configured
+    email forwarding.
+
+You can start with Sieve by writing scripts to the `~/sieve` file of your home
+directory. This will be read and executed on all inbound mail.
+
+You can find some example scripts to get started
+[here](https://doc.dovecot.org/configuration_manual/sieve/examples/).
+
+There are plenty of resources out there to allow for filtering using Sieve
+rules.
+
+## Neomutt via SSH
+
+Previously, before we had configured IMAP, local mail delivery was read with
+`neomutt`.
+
+We have reconfigured `neomutt` to attempt to connect via IMAP, however we
+obviously cannot complete the transaction entirely without user passwords.
+
+For this reason, when attempting to read mail with `neomutt` via SSH you will be
+asked for your IMAP password.
+
+You can enter your password here and you will be able to browse your inbox,
+archives, drafts and sent mail (all are automatically configured to the IMAP
+equivalent).
+
+Whilst strongly advised against, you can set the `imap_pass` attribute in your
```

Comment:
If that was the only option, correct, but given the whole point of this system is that we have IMAP support, I would say we prefer the IMAP solution from a remote host rather than encouraging people to store plaintext passwords. It goes against the spirits of everything else we are doing with security hardening and good practices.